recipes/prettytable/meta.yaml

```diff
@@ -0,0 +1,23 @@
+package:
+  name: prettytable
+  version: "7"
```

Comment:
Actually, I guess the version is too old for me to use.  Let me see if I can create a recipe using the proper version.